src/databox/azext_databox/_params.py

```diff
@@ -0,0 +1,78 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from knack.arguments import CLIArgumentType
+
+
+def load_arguments(self, _):
+    job_name_type = CLIArgumentType(
+        help='The name of the job resource within the specified resource group. job names must be between 3 and 24 '
+             'characters in length and use any alphanumeric and underscore only')
```

Comment:
add `options_list=['--name', '-n']`?

Reply:
Will update it.

---
In reply to: [388084566](https://github.com/Azure/azure-cli-extensions/pull/1344#discussion_r388084566) [](ancestors = 388084566)